contracts/BonusFinalizeAgent.sol

```diff
@@ -35,17 +35,10 @@ contract BonusFinalizeAgent is FinalizeAgent {
   uint public allocatedBonus;
 
   function BonusFinalizeAgent(CrowdsaleToken _token, Crowdsale _crowdsale, uint _bonusBasePoints, address _teamMultisig) {
+    require(address(_crowdsale) != 0 && address(_teamMultisig) != 0);
```

Comment:
Should these be two separate require()s?

Reply:
Thanks for the compliments, changed to two separate require()s as requested.

Reply:
Thanks for the compliments, changed to two separate require()s as requested.